docs/design/specs/Ecma-335-Augments.md

```diff
@@ -1,4 +1,4 @@
-# Known Issues in ECMA-335 CLI Specification 
+# Additions to the ECMA-335 CLI Specification
```

Comment:
https://github.com/dotnet/runtime/blob/master/src/libraries/System.Reflection.Metadata/specs/PE-COFF.md title says "Addendum". Maybe we can do the same here to maintain same style.